packages/pdf.lua

```diff
@@ -108,15 +108,24 @@ end)
 
 SILE.registerCommand("pdf:metadata", function (options, _)
   local key = SU.required(options, "key", "pdf:metadata")
-  local val = SU.required(options, "val", "pdf:metadata")
+  local value
+  if options.val ~= nil then
+    SU.deprecated("\\pdf:metadata[…, val=…]", "\\pdf:metadata[…, value=…]", "0.12.0", "0.13.0")
+    value = SU.required(options, "val", "pdf:metadata")
+  else
+    value = SU.required(options, "value", "pdf:metadata")
+  end
+
+  -- see comment in pdf:bookmark
+  value = SU.utf8_to_utf16be(value)
```

Comment:
Should we do it always, even if the key is ModDate or CreationDate? The PDF specs do not say dates can be UTF16-BE encoded. This might work if viewers/readers are flexible enough, but might not be compliant.

Reply:
Fixed in https://github.com/sile-typesetter/sile/pull/1281/commits/7df0242865c4c4ef44e94bb26d3e61440244d165.